css/index.css

```diff
@@ -143,4 +143,113 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+/* ... */
+.navbar {
+    display: flex;
+    align-items: baseline;
+    justify-content: baseline;
+    width: 100%;
+    padding-bottom: 40px;
+    margin-top: 20px;
+}
+.nav-logo {
+    width: 25%;
+    padding-top: 5px;
+}
+.nav-links {
+    width: 70%;
+    display: flex;
+    justify-content: space-between;
+    text-transform: capitalize;
+}
+.nav-links a {
+    text-decoration: none;
+    color: rgba(0, 0, 0, 0.705);
+}
+.header-img {
+    padding-bottom: 20px;
+}
+.middle-content .boxes .box1 {
+    background-color: teal;
+}
+.middle-content .boxes .box2 {
+    background-color: gold;
+}
+.middle-content .boxes .box3 {
+    background-color: cadetblue;
+}
+.middle-content .boxes .box4 {
+    background-color: coral;
+}
+.middle-content .boxes .box5 {
+    background-color: crimson;
+}
+.middle-content .boxes .box6 {
+    background-color: forestgreen;
+}
+.middle-content .boxes .box7 {
+    background-color: darkorchid;
+}
+.middle-content .boxes .box8 {
+    background-color: hotpink;
+}
+.middle-content .boxes .box9 {
+    background-color: indigo;
+}
+.middle-content .boxes .box10 {
+    background-color: dodgerblue;
+}
+
+/* about page */
+.about-top-content {
+    margin: 0 auto;
+    padding-left: 20px;
+    padding-bottom: 20px;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+  
+
+}
```

Comment:
line 211 and 212 are white, cleaning your code it makes it look more professional. 
Also sometimes you have a white line between your classes, sometimes you don't. It's not technically a problem but it makes your code looks sloppier. Try to be consistent.